src/cmd/run.go

```diff
@@ -57,6 +69,41 @@ func doRun(cmd *cobra.Command, args []string) {
 	client.RunnerUnregister(&runner.Id)
 }
 
+func electLeader() {
+	leaseLockName := "opslevel-runner-leader-lock"
+	leaseLockNamespace := viper.GetString("pod-namespace")
+	podName := viper.GetString("pod-name")
+
+	config, err := pkg.GetKubernetesConfig()
+
+	if err != nil {
+		log.Info().Msgf("Failed to get kubeconfig")
```

Comment:
should we exit here?  If the `GetKubernetesConfig` fails all we'll do is log this info message but continue on with the logic.  This feels wrong